internal/pkg/agent/install/progress_test.go

```diff
@@ -110,7 +110,7 @@ func TestProgress(t *testing.T) {
 
 		rs.Failed()
 
-		require.Equal(t, "step 1 starting... FAILED\n", string(w.buf))
+		require.Regexp(t, regexp.MustCompile(`step 1 starting.{3,} FAILED\n`), string(w.buf))
```

Comment:
Are we sure this one needs to be changed? There is no delay in this test between the step starting and failing so we should expect to see exactly three `.`s.

Reply:
I'm being paranoid, the Go scheduler could stop your thread of execution between the `StepStart` and `Failed`, then OS scheduler puts your process to sleep, and the container run time halts your container for 10 seconds, when it all comes back the ticker has gone off and you get more "." than you expected.

Reply:
Fair enough.